pubspec.yaml

```diff
@@ -5,6 +5,7 @@ repository: https://github.com/srawlins/timezone
 environment:
```

Comment:
Likely needs a version bump and a changelog entry, but @srawlins can comment there